srcpkgs/go-ethereum/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'go-ethereum'
+pkgname=go-ethereum
+version=1.8.18
+revision=1
+build_style=go
+go_import_path="github.com/ethereum/go-ethereum"
+go_package="${go_import_path}/cmd/abigen
+${go_import_path}/cmd/bootnode
+${go_import_path}/cmd/evm
+${go_import_path}/cmd/geth
+${go_import_path}/cmd/puppeth
+${go_import_path}/cmd/rlpdump
+${go_import_path}/cmd/wnode
+${go_import_path}/cmd/swarm"
+short_desc="Full suite of Go Ethereum utilities"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="GPL-3.0-only"
+homepage="https://github.com/ethereum/go-ethereum"
+distfiles="https://${go_import_path}/archive/v${version}.tar.gz"
+checksum=cbab18a733298830c9ed1e19c1ece37edf417fd55ec8f198803048ecc3ffa0b9
+conflicts="geth"
```

Comment:
isn't it better to just have a geth subpackage if there is demand for only geth ?